test/e2e/disability-benefits-helpers.js

```diff
@@ -17,14 +18,38 @@ function completeEvidenceTypeInformation(client, data) {
 }
 
 function completeVAFacilitiesInformation(client, data) {
-  client
-    .selectDropdown('root_treatments_0_treatment_startTreatmentMonth', data.treatments[0].startTreatmentMonth)
-    .selectDropdown('root_treatments_0_treatment_startTreatmentDay', data.treatments[0].startTreatmentDay)
-    .fill('input[name="root_treatments_0_treatment_startTreatmentYear"]', data.treatments[0].startTreatmentYear)
-    .selectDropdown('root_treatments_0_treatment_endTreatmentMonth', data.treatments[0].endTreatmentMonth)
-    .selectDropdown('root_treatments_0_treatment_endTreatmentDay', data.treatments[0].endTreatmentDay)
-    .fill('input[name="root_treatments_0_treatment_endTreatmentYear"]', data.treatments[0].endTreatmentYear)
-    .fill('input[name="root_treatments_0_treatment_treatmentCenterName"]', data.treatments[0].treatmentCenterName);
+  data.treatments.forEach((treatment, i) => {
+    client
+      .waitForElementVisible(`input[name="root_treatments_${i}_treatment_treatmentCenterName"]`, Timeouts.normal)
+      .selectDropdown(`root_treatments_${i}_treatment_startTreatmentMonth`, data.treatments[0].startTreatmentMonth)
+      .selectDropdown(`root_treatments_${i}_treatment_startTreatmentDay`, data.treatments[0].startTreatmentDay)
+      .fill(`input[name="root_treatments_${i}_treatment_startTreatmentYear"]`, data.treatments[0].startTreatmentYear)
+      .selectDropdown(`root_treatments_${i}_treatment_endTreatmentMonth`, data.treatments[0].endTreatmentMonth)
+      .selectDropdown(`root_treatments_${i}_treatment_endTreatmentDay`, data.treatments[0].endTreatmentDay)
+      .fill(`input[name="root_treatments_${i}_treatment_endTreatmentYear"]`, data.treatments[0].endTreatmentYear)
+      .fill(`input[name="root_treatments_${i}_treatment_treatmentCenterName"]`, data.treatments[0].treatmentCenterName);
+    if (i < 1) client.click('.va-growable-add-btn');
```

Comment:
Did you mean `if (i < treatments.length - 1)`?

Reply:
Nice catch, fixed.